src/Workspaces/CoreTest/SymbolKeyTests.cs

```diff
@@ -636,7 +636,7 @@ void goo()
             var xSymbol = testModel.LookupSymbols(position).First(s => s.Name == "x");
 
             // This should not throw an exception.
-            Assert.NotNull(SymbolKey.Create(xSymbol));
+            Assert.NotEqual(default, SymbolKey.Create(xSymbol));
```

Comment:
That never returns null by contract, you can just delete the Assert.NotEqual and leave the inner SymbolKey.Create().